Tasks/AzureMonitorAlerts/azurerestutility.ts

```diff
@@ -0,0 +1,265 @@
+import * as tl from "vsts-task-lib/task";
+import * as Q from "q";
+import * as querystring from "querystring";
+
+import { 
```

Comment:
So what will be a better approach here :
1. exporting the httpClient/restClient object in azure-arm-rest, so that others could use the object 
2. moving all the current code to azure-arm-rest itself